src/super_gradients/training/models/conversion.py

```diff
@@ -129,3 +137,15 @@ def convert_from_config(cfg: DictConfig) -> str:
     out_path = models.convert_to_onnx(model=model, **cfg)
     logger.info(f"Successfully exported model at {out_path}")
     return out_path
+
+
+def onnx_simplify(onnx_path: str, onnx_sim_path: str):
+    """
+    onnx simplifier method, same as `python -m onnxsim onnx_path onnx_sim_path
+    :param onnx_path: path to onnx model
+    :param onnx_sim_path: path for output onnx simplified model
+    """
+    model_sim, check = simplify(model=onnx_path)
+    assert check, "Simplified ONNX model could not be validated"
```

Comment:
🙏  assert in unit tests, if/raise in library code

Why: Assertions can be toggled on or off at will.

https://stefan.sofa-rockers.org/2018/04/16/assertions-and-exceptions/

TLDR: 

> So never use an assert when another method of capturing and raising an error is possible. Always write your code with the assumption that the assertions could disappear at any minute. And if you’re unsure, you can run your program with python -O or by setting PYTHONOPTIMIZE=1 to test if it still works.

Reply:
Completely right, this is something I always comment about as well, it slipped in :sweat_smile: .

Reply:
Anyway, fixed.